CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing
+Hi! 
+
+Thanks for considering participation in WarsawLO project!
+We believe that tools like ours should be crafted by the community! 
+
+We hope that this guide will help you find your own way of participation.
+If you still have questions after reading this, feel free to reach us.
+
+## How can I contribute to this project? :bulb:
+
+### Report bugs :bug:
+If you notice some bug on our page, check if the bug haven't been already reported, and if not please file an issue.
+
+It would be the best if when opening a new issue with bug report you would provide environment information (such as OS & browser versions).
+
+If you are not a technical person or don't want to use GitHub to report a bug, please contact us via email.
+
+### Code :computer:
+> We assume you have basic understanding of how git & GitHub work.
+>  If you are not comfortable with GitHub workflow [this guide](https://github.com/firstcontributions/first-contributions) may be helpful. Check it out and come back to us later :slightly_smiling_face:
+
+We try to hold all code/features-related discussions in issues sections of corresponding repositories ([warsawlo-2.0](https://github.com/WarsawLO/warsawlo-2.0) for front-end or [warsawlo-django](https://github.com/WarsawLO/warsawlo-django) for back-end).
+Just browse the issues and let us know in a comment that you want to handle it. We'll assign you to that issue.
+
+Please keep in mind when contributing:
+* **Use pull request not direct commit**
+* Please first discuss the change you wish to make via issue with the owners of the repository before making a change unless you the issue you're working on is assigned to you.
```

Comment:
Please first discuss the change you wish to make by commenting on a related issue, unless it is assigned to you.